release/blocker_tracker_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+JIRA_TOKEN=""
+SLACK_WEB_HOOK_URL=""
+
+function main() {
+    local blocker_issues_keys=($(curl --request GET --url 'https://liferay.atlassian.net/rest/api/3/search?jql=project%20%3D%20%22PUBLIC%20-%20Liferay%20Product%20Delivery%22%20and%20labels%20%3D%20release-blocker%20and%20status%20%21%3D%20Closed&fields=issuekey' --user "jira-cloud-enterprisereleasehu@liferay.com:<$JIRA_TOKEN>" --header 'Accept: application/json' | jq -r '.issues[].key'))
+
+    cd liferay-portal || (echo "Failed to change directory to liferay-portal"; exit)
+
+    local git_pull_response
+
+    git_pull_response=$(git pull origin master)
+
+    if [[ "${git_pull_response}" == *"Already up to date"* ]]
+    then
+        echo "The master branch hasn't been updated since the last run, so no changes in the blocks have occurred"
+        exit
+    fi
+
+    local not_yet_merged_blockers=""
+
+    for blocker_issue_key in "${blocker_issues_keys[@]}"
+    do
+        local git_log_grep_result
+
+        git_log_grep_result=$(git log --grep="${blocker_issue_key}")
+
+        if [ -z "${git_log_grep_result}" ]
+        then
+            not_yet_merged_blockers+="<https://liferay.atlassian.net/browse/${blocker_issue_key}|${blocker_issue_key}> "
+        fi
+    done
+
+    local slack_message="All blockers are merged"
+
+    if [ -n "${not_yet_merged_blockers}" ]
+    then
+        slack_message+="These blockers still need to be merged: ${not_yet_merged_blockers}"
+    fi
+
+    slack_response_code=$(curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"${slack_message}\"}" "${SLACK_WEB_HOOK_URL}" | jq -r '.ok')
```

Comment:
@kiwm, no need to declare this variable, please do as [here](https://github.com/liferay/liferay-docker/blob/73809f47af89ccb12d0d38ebe4cd0bf333cb0b4e/release/_product.sh#L367) and style as [here](https://github.com/liferay/liferay-docker/blob/master/release/release_gold.sh#L113-L129). :wink: 

In addition, can you please include a error message in case the `curl` fails?

Thanks